app/Events/GameIntervalEvent.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Events;
+
+class GameIntervalEvent extends GameEvent {
+
+    public function broadcastAs(): string {
+        return 'game.interval';
```

Comment:
Deze wordt nergens in het PR gebruikt

Reply:
Klopt, is voor een andere taak. Maar ik had het al aangemaakt voordat ik me dat bedacht.